ts/features/wallet/bancomat/screen/BancomatDetailScreen.tsx

```diff
@@ -15,13 +15,14 @@ import { navigateToOnboardingCoBadgeChooseTypeStartScreen } from "../../onboardi
 import BancomatCard from "../component/bancomatCard/BancomatCard";
 import BancomatInformation from "./BancomatInformation";
 
-type NavigationParams = Readonly<{
+export type BancomatDetailScreenNavigationParams = Readonly<{
+  // TODO: we should use only the id and retrieve it from the store, otherwise we lose all the updates
```

Comment:
❓**question**: I don't think I'm understanding the comment. Is the ID of the payment method? And it means that we should persist it to the store?
Thanks for your patience, if you have a link to some internal docs I'd be happy yo read them independently 😄

Reply:
Yes, is the id of the payment method. Is not reccomanded to pass a whole object through the navigation parameters, especially if it is present in the store and can receive updates. You can find further details in https://reactnavigation.org/docs/5.x/params#what-should-be-in-params.